docs/datasets/stack.ipynb

```diff
@@ -0,0 +1,362 @@
+{
```

Comment:
<p>Here the actual treatment is a bit more complex in 3D. In particular regarding regions fully excluded (generally because of offset cut).</p><p>Maybe an explanation for what is going on in 3D in needed somewhere.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gammapy/gammapy/pull/3282/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>